README.md

```diff
@@ -45,6 +46,7 @@ docker run --name "dd-agent-build-$PLATFORM" \
   -e AGENT_BRANCH=$AGENT_BRANCH \
   -e AGENT_VERSION=$AGENT_VERSION \
   -e TRACE_AGENT_BRANCH=$TRACE_AGENT_BRANCH \
+  -e PROCESS_AGENT_BRANCH=$PROCESS_AGENT_BRANCH \
```

Comment:
you'll have to create PRs on the:
- https://github.com/DataDog/docker-dd-agent-build-deb-x64
- https://github.com/DataDog/docker-dd-agent-build-rpm-x64
- https://github.com/DataDog/docker-dd-agent-build-rpm-suse-x64

on the `circle.yml` file to add support for the flag.